lib/resque.rb

```diff
@@ -298,6 +298,22 @@ def to_s
   # The 'inline' is false Resque jobs will be put in queue regularly.
   alias :inline? :inline
 
+  # if `reconnect_redis_per_job` is true (default), each job will get a
+  # fresh Redis connection. if false, Resque workers will instantiate a
+  # single Redis connection and share it across forks with individual workers.
+  # a false setting tends to be more efficient in terms of connection overhead,
+  # but may not work with all client libraries.
```

Comment:
Which client libraries does it not work with? Maybe we could be more explicit here and mention we only recommend using this with redis-rb?

Reply:
I only tested with `redis-rb`. Agree that calling that out ("we're confident it works for `redis-rb`; you should test thoroughly with other libraries") is a good move.